### ISSUE TITLE
Cannot find image_label for pre-training

### ISSUE THREAD

**Issue by yikaiw** [microsoft/Oscar#85]
I followed [https://github.com/microsoft/Oscar/blob/master/VinVL_DOWNLOAD.md#pre-exacted-image-features](url) to prepare image features, and followed [https://github.com/microsoft/Oscar/blob/master/VinVL_MODEL_ZOO.md#oscarplus-pretraining](url) for pre-training. But I cannot find where are the **image labels** for the pre-training datasets, e.g. COCO, flickr30k, GCA. 

As shown in [https://biglmdiag.blob.core.windows.net/vinvl/pretrain_corpus/coco_flickr30k_googlecc_gqa_sbu_oi_x152c4big2exp168.yaml](url), we need to prepare image_label_path.
`corpus: coco_flickr30k_gqa_googlecc_sbu_oi
corpus_file: coco_flickr30k_googlecc_gqa_sbu_oi.tsv
image_label_path:
  coco: X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/coco
  flickr30k: X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/flickr30k
  gqa: X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/gqa
  googlecc: X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/googlecc
  sbu: X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/sbu
  oi: X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/oi
image_feature_path:
  coco: vinvl/image_features/coco_X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/model_0060000
  flickr30k: vinvl/image_features/flickr30k_X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/model_0060000
  gqa: vinvl/image_features/gqa_X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/model_0060000
  googlecc: vinvl/image_features/googlecc_X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/model_0060000
  sbu: vinvl/image_features/sbu_X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/model_0060000
  oi: vinvl/image_features/oi_X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2/model_0060000`

To be specific, there is no guidance to download or generate **predictions_gt.tsv** and **QA_fileB.tsv** files, which are needed for pre-training in [https://github.com/microsoft/Oscar/blob/master/oscar/datasets/oscar_tsv.py#L383-L385](url).

**Comment by yikaiw** [comment]
To be specific, there is no guidance to download or generate **predictions_gt.tsv** and **QA_fileB.tsv** files, which are needed for pre-training in [https://github.com/microsoft/Oscar/blob/master/oscar/datasets/oscar_tsv.py#L383-L385](url).

**Comment by biluoyuebai** [comment]
I had the same problem, but I found these files under this URL.
https://biglmdiag.blob.core.windows.net/vinvl/pretrain_corpus/X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2
You can download it with
`<path-to-azcopy> copy "https://biglmdiag.blob.core.windows.net/vinvl/pretrain_corpus/X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2" . --recursive`
Hope this can help you.

**Comment by yikaiw** [comment]
> I had the same problem, but I found these files under this URL.
> https://biglmdiag.blob.core.windows.net/vinvl/pretrain_corpus/X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2
> You can download them with
> `<path-to-azcopy> copy "https://biglmdiag.blob.core.windows.net/vinvl/pretrain_corpus/X152C4_frcnnbig2_exp168model_0060000model.roi_heads.nm_filter_2_model.roi_heads.score_thresh_0.2" . --recursive`
> Hope this can help you.

Great, many thanks!

**Comment by wangxiao5791509** [comment]
@yikaiw   Hi, did you run this code smoothly? where to download the "**coco_flickr30k_googlecc_gqa_sbu_oi_x152c4big2exp168.yaml**"??? The link provided by @biluoyuebai  is only the label files, but not the needed *.yaml file. Thanks for your help.
